src/cmd_line.rs

```diff
@@ -146,7 +146,7 @@ pub fn handle_command_line_arguments() -> Result<(), String> {
             || std::env::var("NeovideMultiGrid").is_ok()
             || matches.is_present("multi_grid"),
         remote_tcp: matches.value_of("remote_tcp").map(|i| i.to_owned()),
-        disowned: matches.is_present("disowned"),
+        nofork: matches.is_present("nofork") || matches.is_present("verbosity"),
```

Comment:
Can you explain the verbosity term? Maybe add a comment? Its not obvious to me why its added

Reply:
verbosity is not added, it was already there.

what is added, is using `-v -vv -vvv ...` implies `--nofork`, becouse verbosity without nofork is useless (as process detaches from terminal and we don't see logs)

Reply:
Do you thin comment in this place?

It is explained in `help` for `verbosity` itself. Maybe message could be improved, any suggestions?

Reply:
Nah its all good. This makes sense